pkg/kv/kvserver/client_split_test.go

```diff
@@ -496,11 +496,16 @@ func TestStoreRangeSplitAtRangeBounds(t *testing.T) {
 	}
 	replCount := store.ReplicaCount()
 
-	// An AdminSplit request sent to the end of the old range
-	// should fail with a RangeKeyMismatchError.
-	_, pErr := kv.SendWrappedWith(ctx, store, h, args)
-	if _, ok := pErr.GetDetail().(*roachpb.RangeKeyMismatchError); !ok {
-		t.Fatalf("expected RangeKeyMismatchError, found: %v", pErr)
+	// An AdminSplit request sent to the end of the old range should be re-routed
+	// to the start of the new range, succeeding but no new ranges should be created
```

Comment:
s/but no ../but without creating any new ranges, also missing period at the end.

Reply:
Updated.